gobreaker.go

```diff
@@ -18,6 +19,25 @@ const (
 	StateOpen
 )
 
+// ErrTooManyRequests is returned when the CB state is half open and the requests count is over the cb maxRequests
+var ErrTooManyRequests = errors.New("too many requests")
+
+// ErrCircuitOpen is returned when the CB state is open
+var ErrCircuitOpen = errors.New("circuit breaker is open")
```

Comment:
I think ErrOpenState is better than ErrCircuitOpen because the package name gobreaker already implies CircuitBreaker.